code/src/App.js

```diff
@@ -1,12 +1,61 @@
-import React from 'react'
-import data from './data.json'
+import React from 'react';
+import data from './data.json';
+import json from './stretch-goal.json';
+import { Header } from './Components/Header.js';
+import { Album } from './Components/Album.js';
+import { Playlists } from './Components/Playlists.js';
+import { Playlistheader } from './Components/Playlistheader.js';
+import { HeaderSingles } from './Components/HeaderSingles.js';
+import { Singles } from './Components/Singles.js';
```

Comment:
There is a convention that says that we should divide outer dependencies (those located in node_modules) from local dependencies (all local files that we created - components, functions, data such as arrays) by one empty line. Moreover, style.css should be separated as well and located below local dependencies.